### PR TITLE
3.2.0: minor nits

### PR DESCRIPTION
**Pull request by ralfhandl**
Port of
* #4014

Builds on and contains
* #4025

### CODE REVIEW

File: versions/3.2.0.md [hunk 1]
```diff
@@ -145,82 +97,83 @@ For example, if a field has an array value, the JSON array representation will b
...
 
 Implementations MAY support complete-document parsing in any of the following ways:
 
 * Detecting OpenAPI or JSON Schema documents using media types
 * Detecting OpenAPI documents through the root `openapi` property
 * Detecting JSON Schema documents through detecting keywords or otherwise successfully parsing the document in accordance with the JSON Schema specification
-* Detecting a document containing a referenceable Object at its root based on the expected type of the reference
-* Allowing users to configure the type of documents that might be loaded due to a reference to a non-root Object
+* Detecting a document containing a referenceable object at its root based on the expected type of the reference
+* Allowing users to configure the type of documents that might be loaded due to a reference to a non-root object
```
[the remark this answers is not in the document]
Comment (ralfhandl) [reply versions/3.2.0.md:140]: How about using "OAS-defined Object" here? That would be more noticeable than a single upper-case letter.

**Review by handrews — CHANGES_REQUESTED** [review]
[Same comment as before, was supposed to be a "Request Changes" not "Approve", my apologies for hitting the wrong button]

@ralfhandl can you re-do this with the tables not expanded since we agreed on that for the other branches? That will keep the change history so much less polluted on this branch.

Ideally I'd prefer not collapsing the spaces after periods, but I'm not going to my heels in on it (but it does create more noise and makes it harder to find sentence changes in a fixed-width editor).

Also please fix the two places where "Object" should be capitalized- I can also get it when porting the addition of the "Object" definition, but it would be better to not add change history noise there either if we can avoid it.

**Comment by ralfhandl** [comment]
Can do that after #4025 is merged as this builds on it.

**Comment by handrews** [comment]
@ralfhandl I've approved the previous PR in the sequence, so once you merge and rebase I'll look over this one.

**Comment by ralfhandl** [comment]
@handrews & @miqui 

> once you merge and rebase I'll look over this one

Merged #4025 and rebased this one, please review

File: versions/3.2.0.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ralfhandl) [reply versions/3.2.0.md:140]: I'll fix this in a follow-up sync PR, this is the direct port of #4014.

We have made changes to 3.1.1 in the meantime that need syncing anyway.

File: versions/3.2.0.md [hunk 2]
```diff
@@ -314,7 +314,7 @@ This is the root object of the [OpenAPI document](#openapi-document).
...
 | <a name="oas-components"></a>components | [Components Object](#components-object) | An element to hold various schemas for the document. |
-| <a name="oas-security"></a>security | [[Security Requirement Object](#security-requirement-object)] | A declaration of which security mechanisms can be used across the API. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. Individual operations can override this definition. To make security optional, an empty security requirement (`{}`) can be included in the array. |
+| <a name="oas-security"></a>security | [[Security Requirement Object](#security-requirement-object)] | A declaration of which security mechanisms can be used across the API. The list of values includes alternative Security Requirement Objects that can be used. Only one of the Security Requirement Objects need to be satisfied to authorize a request. Individual operations can override this definition. To make security optional, an empty security requirement (`{}`) can be included in the array. |
```
Comment (miqui) [inline versions/3.2.0.md:317]: >To make security optional
Should be: To make security explicitly optional

To conform to https://github.com/OAI/OpenAPI-Specification/pull/4059/files

**Review by miqui — COMMENTED** [review]
@ralfhandl checkout my comment

File: versions/3.2.0.md (hunk 2 above)
Comment (ralfhandl) [reply versions/3.2.0.md:317]: @miqui I'd rather port this separately, similar to the 3.1.1 port in
* #4064

Once 3.1.1 is published I'll do another diff between 3.1.1 and 3.2.0 to catch all accumulated differences.

Would that work for you?

Plus, this PR is the basis that the next PR port
* #4035 

is based on.

File: versions/3.2.0.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ralfhandl) [reply versions/3.2.0.md:140]: @handrews There are more changes in 
* #4052 

that need to be ported to 3.2.0, for example the definition of "Object" and adjustment of headlines.

If we want to collapse all diffs into one large PR, we can do that by closing this PR and enhancing 
* #4035

Which way to go?

[the remark this answers is not in the document]
Comment (ralfhandl) [reply versions/3.2.0.md:140]: Created
* #4066

File: versions/3.2.0.md (hunk 2 above)
Comment (ralfhandl) [reply versions/3.2.0.md:317]: Created 
* #4066

**Review by miqui — APPROVED** [review]

**Review by handrews — APPROVED** [review]

**Pull request merged**